README.md

```diff
@@ -194,6 +194,7 @@ Resource name is written in the plural form.
 |---------|-----------|-------|-------|
 |`fields=`|Fields to receive|All fields|`name,emails.address`|
 |`preloads=`|Nested resources to preload|(empty)|`emails,profile`|
+|`pretty`|Formats JSON output|`false`|Not required value|
```

Comment:
Hmm...

- Description: Prettify JSON response
- Example: `true`

would be better.